content/en/service_management/on-call/automations.md

```diff
@@ -13,7 +13,7 @@ further_reading:
 
 Automations allow you to trigger actions based on events in On-Call. There are two types:
 - **Handover automations**: Triggered automatically when an on-call shift changes.
-- **[Workflow automations][1]**: A low-code/no-code solution that lets you define actions based on various on-call activities.
+- **[Workflow automations][1]**: A low-code/no-code solution that can include on-call actions (such as paging, or updating schedules) within automated workflows. For all On-Call related actions, see [here][5].
```

Comment:
```suggestion
- **[Workflow automations][1]**: A low-code/no-code solution that can include on-call actions (such as paging or updating schedules) within automated workflows. For all on-call related actions, see the [Actions Catalog][5].
```

Reply:
I recognize that casing for `on-call` is somewhat inconsistent on this page, but it's mostly _not_ capitalized. If the intention is to refer to Datadog's on-call offering as a whole, I would suggest updating the second `On-Call`  to `Datadog On-Call` for clarity.